### DISCUSSION TITLE
SEB browswer does not open

### DISCUSSION THREAD

**Discussion by KarthikTMK** [SafeExamBrowser/seb-win-refactoring#309] [Q&A]
SEB browser does not wrok , throws a config file error
![IMG_20211223_143101](https://user-images.githubusercontent.com/96572856/147243444-92505fdb-c82e-4b54-a51e-6b6215af10f7.jpg)

**Comment by danschlet** [comment]
Well, obviously the config file is wrong. So ask the person you got the config file from, there seems to be an error in that file. Also they should not send you config files, but place it on a web server and provide a sebs:// link to that file. Such a sebs:// link will automatically open in SEB and SEB downloads that config file.

**Comment by KarthikTMK** [comment]
Thanks for this

I had installed the SEB V 2.4.1 in Apr directly from the website, no config
files etc

It worked just fine

I had uninstalled it a few more later and now when I am trying to install
the same or the latest version this is the issue i am getting
